### ISSUE TITLE
Improve development turnaround by loading index.min.js from the disk

### ISSUE THREAD

**Issue by richorama** [OrleansContrib/OrleansDashboard#173]
As suggested by @seniorquico 

Developing the dashboard can be slow, as you have to restart the C# test host after every change to the JS.

If a index.min.js file exists on the disk, we could pick this up in preference to the embedded file, and save the test host restart.

This time could be further improved if a technology such as webpack was used to build the minified javascript incrementally.

**Comment by seniorquico** [comment]
It looks like [DashboardMiddleware](https://github.com/OrleansContrib/OrleansDashboard/blob/e5826095a74e5545f87d23d425b5a9211b9711e2/OrleansDashboard/DashboardMiddleware.cs#L279-L284) already tries to load "index.min.js" from disk. I'm now guessing this isn't working due to an issue with my test process's current directory or something. I'll try to step through the middleware to see if I can identify the cause in my test environment.

Thinking about a webpack-based build, [ASP.NET's SPA services](https://docs.microsoft.com/en-us/aspnet/core/client-side/spa/react?view=aspnetcore-2.1) include a proxy middleware that could be paired with webpack's dev server. Based on some quick searches, it sounds like the proxy middleware even propagates webpack's live-reload/HRM signals. This could be added to the self-host config like:

```c#
if (env.IsDevelopment())
{
    app.UseSpa(spa =>
    {
        spa.UseProxyToSpaDevelopmentServer("http://localhost:3000");
    });
}
```

I'll admit, I don't have any experience with ASP.NET's SPA services packages. Feedback welcome.

**Comment by richorama** [comment]
closed by #178 & #175
